src/components/SubscriptAvatar.tsx

```diff
@@ -1,46 +1,59 @@
 import React, {memo} from 'react';
-import PropTypes from 'prop-types';
 import {View} from 'react-native';
-import _ from 'underscore';
-import lodashGet from 'lodash/get';
+import {SvgProps} from 'react-native-svg';
+import {ValueOf} from 'type-fest';
 import styles from '../styles/styles';
 import themeColors from '../styles/themes/default';
 import Avatar from './Avatar';
 import CONST from '../CONST';
 import * as StyleUtils from '../styles/StyleUtils';
-import avatarPropTypes from './avatarPropTypes';
 import UserDetailsTooltip from './UserDetailsTooltip';
 
-const propTypes = {
+type SubAvatar = {
+    /** Avatar source to display */
+    source: React.FC<SvgProps> | string;
```

Comment:
I think @fabioh8010 in one of his PR created reusable type from it , so I think you can use it also

Reply:
@kubabutkiewicz I've found what you're talking about, but it's not merged yet.
@fabioh8010  should I wait for [that PR](https://github.com/Expensify/App/pull/30328/files) to be merged first then?

Reply:
Yes, I created [here](https://github.com/Expensify/App/pull/30328/files#diff-6bde6d64d118fa3586ce779a52e1fd38220d561bc4ec71cc60881f812a4b9d94R11). I think you can create in the same place to avoid hold more PRs on anothers